src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializer.java

```diff
@@ -456,14 +456,19 @@ protected Object _deserializeUsingPropertyBased(final JsonParser p, final Deseri
                         bean = wrapInstantiationProblem(e, ctxt);
                     }
                     if (bean == null) {
-                        return ctxt.handleInstantiationProblem(handledType(), null,
-                                _creatorReturnedNullException());
+                        // [databind#4938] Since 2.19, Allow returning `null` from creator, but
+                        // only after doing the regular handling of buffered properties
```

Comment:
No,not quite -- cannot do regular handling since we have no `bean`. Instead, call a new method which will handle skipping, passing till the end.